File: iOS/SDK/smartsip-sdk/smartsip-sdk/CallInfo.swift
//
//  CallCredentials.swift
//  smartsip-sdk
//
//  Created by Franz Iacob on 09/01/2026.
//

public struct CallInfo {
    public let sessionId: String
    public let domain: String
    public let port: Int
    public let username: String
    public let password: String
    public var callerFullName: String?
}

File: Android/smartsip-sdk/src/main/java/cc/smartconnect/smartsip_sdk/private/CallInfo.kt
package cc.smartconnect.smartsip_sdk.private

/**
 * CallInfo.kt
 * smartsip-sdk
 *
 * Created by Franz Iacob on 21/01/2026.
 * * Internal data class representing the essential parameters for a SIP session.
 */
internal data class CallInfo(
    val sessionId: String,
    val domain: String,
    val port: Int,
    val username: String,
    val password: String,
    val destination: String?,

    // Mutable as it may be updated during the call setup lifecycle
    var callerFullName: String?
)